packages/docs/docs/getting-started/faqs.md

```diff
@@ -0,0 +1,28 @@
+---
+
+title: FAQs
+---
+
+# ❓FAQs
+
+What is Unirep?
+
+* UniRep is an anonymous reputation protocol. The protocol allows developers to build applications that associate data with anonymous users. 
+
+What is an epoch key?
+
+* Epoch Keys are how a user stays anonymous while receiving data. An epoch key is a temporary anonymous identity or persona that a user uses to interact with a Unirep application across epochs that are renewed each epoch.
+
+What is the architecture of a Unirep application?
+
+* A Unirep application will have the architecture of web app <-> relay <-> blockchain. 
+
+How can I get started building on Unirep?
+* Refer to our 'Getting Started' section on how to bootstrap a UniRep application.
+
+
+What is an Attester?
+* An Attester, technically, is the smart contract you will create as part of building out the application. The smart contract will be responsible for, literally, attesting the data to the anonymous user. 
```

Comment:
a smart contract or an EOA (Externally-owned account)
Also you can give example like the platform like Github is the attester that is responsible for managing stars and followers or Airbnb is an attester that is not taking part in giving reputation but it sets rules for users to receive reputation.